index.d.ts

```diff
@@ -69,6 +69,13 @@ declare namespace electronDl {
 		@default true
 		*/
 		readonly showBadge?: boolean;
+		
+		/**
+		Allows downloaded files to overwrite files with the same name in the directory they are saved to.
+		
```

Comment:
This should also describe what happens by default, that it adds a number to the filename.